app/src/main/res/layout/main_fragment.xml

```diff
@@ -8,78 +8,76 @@
     tools:context=".UI.MainFragment">
 
 
-        <AutoCompleteTextView
-            android:id="@+id/rcpSearch"
+        <LinearLayout
+            android:id="@+id/linearLayout"
             android:layout_width="0dp"
             android:layout_height="wrap_content"
-
-            android:layout_marginTop="41dp"
-            android:hint="Select a recipe"
-
+            android:layout_marginStart="10dp"
+            android:layout_marginLeft="10dp"
+            android:layout_marginEnd="10dp"
+            android:layout_marginRight="10dp"
+            android:gravity="center"
+            android:orientation="horizontal"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toTopOf="parent"
-            android:text="Select a Recipe"
-            tools:layout_editor_absoluteX="0dp"
-            android:gravity="center_horizontal" />
+            app:layout_constraintTop_toTopOf="parent">
```

Comment:
Adding in a linear layout helped fix the problem where the buttons were running off-screen on some of the devices I tested your app on.